src/action_animations.jl

```diff
@@ -486,3 +486,42 @@ function _change(video, object, action, rel_frame, s)
     val = get_interpolation(action, rel_frame)
     object.change_keywords[s] = val
 end
+
+"""
+morph() to be used with Action, when an animation form Animations.jl is
```

Comment:
Before merge: proper docstring with the function definition indented is needed

Reply:
yes , will do